src/translation/translation_nb_NO.ts

```diff
@@ -2771,7 +2771,7 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../serverdlg.cpp" line="870"/>
         <source>ERROR</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Feil</translation>
```

Comment:
As above: probably this should be all caps.

Reply:
I am of the opinion that nothing should ever be all caps.

https://developer.gnome.org/hig/guidelines/typography.html "Do not capitalize every letter (all caps)."
https://developers.google.com/style/capitalization (says sentence case).
https://develop.kde.org/hig/style/writing/capitalization/ doesn't mention even doing all caps.